mlir/test/Dialect/Quant/Bytecode/types.mlir

```diff
@@ -70,8 +70,8 @@ module @parseUniformPerAxisMixed attributes {
.706000e-01,-6.627000e-01,-6.549000e-01,-6.471000e-01,-0.63919999999999999,-0.63139999999999996,-6.235000e-01,-6.157000e-01,-6.078000e-01,-6.000000e-01,-5.922000e-01,-5.843000e-01,-5.765000e-01,-5.686000e-01,-5.608000e-01,-5.529000e-01,-5.451000e-01,-5.373000e-01,-5.294000e-01,-5.216000e-01,-5.137000e-01,-5.059000e-01,-4.980000e-01,-4.902000e-01,-4.824000e-01,-4.745000e-01,-4.667000e-01,-4.588000e-01,-4.510000e-01,-4.431000e-01,-4.353000e-01,-4.275000e-01,-4.196000e-01,-4.118000e-01,-4.039000e-01,-3.961000e-01,-3.882000e-01,-3.804000e-01,-3.725000e-01,-3.647000e-01,-3.569000e-01,-3.490000e-01,-3.412000e-01,-3.333000e-01,-3.255000e-01,-3.176000e-01,-3.098000e-01,-3.020000e-01,-2.941000e-01,-2.863000e-01,-2.784000e-01,-2.706000e-01,-2.627000e-01,-2.549000e-01,-2.471000e-01,-2.392000e-01,-2.314000e-01,-2.235000e-01,-2.157000e-01,-2.078000e-01,-2.000000e-01,-1.922000e-01,-1.843000e-01,-1.765000e-01,-1.686000e-01,-1.608000e-01,-1.529000e-01,-1.451000e-01,-1.373000e-01,-1.294000e-01,-1.216000e-01,-1.137000e-01,-1.059000e-01,-9.800000e-02,-9.020000e-02,-8.240000e-02,-0.074499999999999997,-0.066699999999999995,-5.880000e-02,-5.100000e-02,-4.310000e-02,-3.530000e-02,-2.750000e-02,-1.960000e-02,-1.180000e-02,-3.900000e-03,3.900000e-03,1.180000e-02,1.960000e-02,2.750000e-02,3.530000e-02,4.310000e-02,5.100000e-02,5.880000e-02,0.066699999999999995,0.074499999999999997,8.240000e-02,9.020000e-02,9.800000e-02,1.059000e-01,1.137000e-01,1.216000e-01,1.294000e-01,1.373000e-01,1.451000e-01,1.529000e-01,1.608000e-01,1.686000e-01,1.765000e-01,1.843000e-01,1.922000e-01,2.000000e-01,2.078000e-01,2.157000e-01,2.235000e-01,2.314000e-01,2.392000e-01,2.471000e-01,2.549000e-01,2.627000e-01,2.706000e-01,2.784000e-01,2.863000e-01,2.941000e-01,3.020000e-01,3.098000e-01,3.176000e-01,3.255000e-01,3.333000e-01,3.412000e-01,3.490000e-01,3.569000e-01,3.647000e-01,3.725000e-01,3.804000e-01,3.882000e-01,3.961000e-01,4.039000e-01,4.118000e-01,4.196000e-01,4.275000e-01,4.353000e-01,4.431000e-01,4.510000e-01,4.588000e-01,4.667000e-01,4.745000e-01,4.824000e-01,4.902000e-01,4.980000e-01,5.059000e-01,5.137000e-01,5.216000e-01,5.294000e-01,5.373000e-01,5.451000e-01,5.529000e-01,5.608000e-01,5.686000e-01,5.765000e-01,5.843000e-01,5.922000e-01,6.000000e-01,6.078000e-01,6.157000e-01,6.235000e-01,0.63139999999999996,0.63919999999999999,6.471000e-01,6.549000e-01,6.627000e-01,6.706000e-01,6.784000e-01,6.863000e-01,6.941000e-01,0.70199999999999996,7.098000e-01,7.176000e-01,7.255000e-01,7.333000e-01,7.412000e-01,7.490000e-01,7.569000e-01,7.647000e-01,7.725000e-01,0.78039999999999998,7.882000e-01,7.961000e-01,8.039000e-01,8.118000e-01,8.196000e-01,8.275000e-01,8.353000e-01,0.84309999999999996,8.510000e-01,8.588000e-01,8.667000e-01,8.745000e-01,8.824000e-01,8.902000e-01,8.980000e-01,9.059000e-01,0.91369999999999995,0.92159999999999997,9.294000e-01,9.373000e-01,9.451000e-01,9.529000e-01,0.96079999999999998,9.686000e-01,9.765000e-01,0.98429999999999995,0.99219999999999997,1.000000e+00}:9.987200e-01:127>
```

Comment:
this case is just for testing a generic type, but in our specific npu scenario, when palletization is enabled and quant.quantile types are in use, the storage type is at most 4 bits wide, so the quantile LUT stored within the type has at most only 16 elements